### ISSUE TITLE
Change colours of markerlegend='on' 

### ISSUE THREAD

**Issue by SAlbertema** [PeterRochford/SkillMetrics#13]
L.S., 

When markerlegend='on' in sm.taylor_diagram(), by default the markers get a blue, red and green colour. How do I modify these colours? 

Secondly, it seems like that markerlegend also sorts the constructed Taylor points. In my case it is sorted for dh: 1000, dh: 250 and dh: 500 which is fine, however:
- 'dh: 250, dt: 0:02:30, R: 50000' is labeled blue while it must be red,
- 'dh: 500, dt: 0:02:30, R: 50000' is labeled red while it must be green,
- 'dh: 500, dt: 0:02:30, R: 70000' is labeled red while it must be green.

I hopy anybody can help me with these two items.

**Comment by PeterRochford** [comment]
I have added a 'markers" option to the SkillMetrics package in version 1.2.2 that I hope to release soon. This allows the user to individually control of the marker label, label color, symbol, size, face color, and edge color via specifications given via a dictionary. Refer to the examples [Target 11](https://github.com/PeterRochford/SkillMetrics/wiki/Examples:-Target-Diagrams#target-diagram-example-11) and [Taylor 15](https://github.com/PeterRochford/SkillMetrics/wiki/Examples:-Taylor-Diagrams#taylor-diagram-example-15).  

I believe that I've also already addressed the markerlegend sort issue. Let me know if otherwise.
